go.mod

```
@@ -114,6 +114,64 @@ require (
 	sigs.k8s.io/controller-tools v0.3.1-0.20200617211605-651903477185
 )
 
+exclude (
+	// https://msazure.visualstudio.com/AzureRedHatOpenShift/_workitems/edit/9555522
```

```suggestion
	// exclude github.com/hashicorp/vault < v1.5.1: https://nvd.nist.gov/vuln/detail/CVE-2020-16251
```

^ would be more meaningful